targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L475xG/device/TOOLCHAIN_ARM_STD/stm32l475xx.sct

```diff
@@ -61,6 +61,7 @@ LR_IROM1 MBED_APP_START MBED_APP_SIZE  {    ; load region size_region
   }
   RW_IRAM1 MBED_RAM0_START MBED_RAM0_SIZE-Stack_Size  { ; RW data 96k L4-SRAM1
    .ANY (+RW, +Last)
+   lwip* (+ZI)
```

Comment:
ARM compiler supports expansion of RW and ZI in multiple banks, please try below solution

RW_IRAM1 MBED_RAM0_START MBED_RAM0_SIZE-Stack_Size  { ; RW data 96k L4-SRAM1
   .ANY (+RW +ZI)
  }
 ; Total: 98 vectors = 392 bytes (0x188) to be reserved in RAM
  RW_IRAM2 (0x10000000+0x188) (0x08000-0x188)  {  ; ZI data 32k L4-ECC-SRAM2
   .ANY (+RW +ZI)
  }